package.json

```diff
@@ -23,7 +23,9 @@
     "path": "^0.12.7",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
+    "react-redux": "^7.2.8",
     "react-router-dom": "^6.3.0",
+    "redux": "^4.1.2",
```

Comment:
Install the latest redux, ignore if this version is the latest.

Reply:
The lastes version is 4.1.2 which we are using currently